web/src/pages/settings/Settings.tsx

```diff
@@ -0,0 +1,111 @@
+import Head from 'next/head';
+import React, { useEffect } from 'react';
+import {
+  AppBar,
+  Avatar,
+  Box,
+  Button,
+  makeStyles,
+  Paper,
+  TextField,
+  Theme,
+  Typography
+} from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+import { useDispatch } from 'react-redux';
+
+import { NavBar } from '../../shared/components/NavBar';
+import { getProductsThunk } from '../../redux/thunk/thunks';
+
+export const Settings = () => {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getProductsThunk());
+  }, []);
```

Comment:
This and line 15 should be removed in your next commit since you shouldn't be dispatching the products thunk here.